pyproject.toml

```diff
@@ -1,3 +1,36 @@
+
+[project]
+name = "belt_nlp"
+version = "0.0.4"
+authors = [
+  { name="Michał Brzozowski", email="michal.brzozowski@mim.ai" },
+  { name="Marek Wachnicki", email="marek.wachnicki@mim.ai" },
+]
+description = "BERT classification model for processing texts longer than 512 tokens."
+keywords = ["nlp", "natural-language-processing", "text-classification", "transformers", "transfer-learning", "bert"]
```

Comment:
We can extend description and keywords for better positioning.
I propose adding `BELT (BERT For Longer Texts)` name to the description. There is the whole readme included so I think it's enough.
For both PyPI and GitHub, we can add keywords such as `roberta` and some proposed by GitHub - `machine-learning`. `deep-learning`, `pytorch`.